desc/compute/_core.py

```diff
@@ -1518,7 +1518,7 @@ def _alpha_r(params, transforms, profiles, data, **kwargs):
     data=["theta_PEST_t", "phi_t", "iota"],
 )
 def _alpha_t(params, transforms, profiles, data, **kwargs):
-    data["alpha_t"] = data["theta_PEST_t"] + data["iota"] * data["phi_t"]
+    data["alpha_t"] = data["theta_PEST_t"] - data["iota"] * data["phi_t"]
```

Comment:
Should phi_t = 0? So it shouldn't change any of the results.

Reply:
Yes it's zero until we introduce the nonzero toroidal stream function. I just caught this as I'm working down the dependencies to see why https://github.com/PlasmaControl/DESC/pull/1003/files#r1641469834 fails.
(The reason the (binary) png file for the effective ripple changed in this commit is likely because I replaced `1/pitch` with `jnp.reciprocal(pitich)` since the Python array library api developers claim it reduces unnecessary data movement between gpu/cpu).